meson.build

```diff
@@ -67,6 +66,16 @@ if openpower_support
     fwupdate_sources += [
         'src/image_openpower.cpp',
     ]
+elif host_image_type == 'intel-x86'
+    conf.set('INTEL_X86_SUPPORT', true)
```

Comment:
Or maybe `X86_FLASH_SUPPORT` ?
It's not about X86 as such, it's about existence of connection between the BMC and the BIOS Flash chip, as well as about existence of the means to restart the PCH.

Reply:
Renamed to `INTEL_C62X_SUPPORT` as we decided it in the chat.